File: ischia/src/main/java/cz/fb/manaus/ischia/BackLoserAdviser.java
package cz.fb.manaus.ischia;

import cz.fb.manaus.reactor.betting.proposer.PriceProposer;
import cz.fb.manaus.reactor.betting.proposer.ProposerAdviser;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Component;

import java.util.List;

@BackLoserBet
@Component
public class BackLoserAdviser extends ProposerAdviser {

    @BackLoserBet
    @Autowired
    public BackLoserAdviser(List<PriceProposer> proposers) {
        super(proposers);
    }
}

File: ischia/src/main/java/cz/fb/manaus/ischia/BackLoserAdviser.kt
package cz.fb.manaus.ischia

import cz.fb.manaus.reactor.betting.proposer.PriceProposer
import cz.fb.manaus.reactor.betting.proposer.ProposerAdviser
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Component

@BackLoserBet
@Component
class BackLoserAdviser @BackLoserBet @Autowired constructor(proposers: List<PriceProposer>) : ProposerAdviser(proposers)
